src/components/SingleArticle/AddCommentForm.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { handleCommentPost } from "./article-click-handlers";
+import "./AddCommentForm.css"
+
+function AddCommentForm({ article_id, user, setComments }) {
+    const [isCommentSubmitted, setIsCommentSubmitted] = useState(false);
+    const [commentInput, setCommentInput] = useState("");
+    const [isSubmitLoading, setIsSubmitLoading] = useState(false);
+
+    return (
+        <form
+            onSubmit={(e) =>
+                handleCommentPost(
+                    e,
+                    setIsCommentSubmitted,
+                    setIsSubmitLoading,
+                    setCommentInput,
+                    article_id,
+                    user,
+                    commentInput,
+                    setComments
+                )
+            } id="add-comment-form"
+        >
+            <label htmlFor="add-comment">Type your comment here:</label>
+            <textarea
+                id="add-comment-input"
+                onChange={(e) => setCommentInput(e.target.value)}
+                value={commentInput}
+            />
+            {isCommentSubmitted && isSubmitLoading ? (
+                <p>Please wait...</p>
```

Comment:
If the user is offline this line will be displayed, we should instead try to let them know that there was an issue with the posting of the comment 🙂

Reply:
Noticed this also, should be fixed within the next pull request